stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/TypeAdapterGenerator.java

```diff
@@ -78,11 +79,10 @@ public TypeSpec getTypeAdapterSpec() {
                 .addModifiers(Modifier.PUBLIC)
                 .addParameter(Gson.class, "gson");
 
-        TypeSpec.Builder adapterBuilder =
-                TypeSpec.classBuilder(mInfo.getTypeAdapterClassName())
-                        .addModifiers(Modifier.PUBLIC, Modifier.FINAL)
-                        .superclass(ParameterizedTypeName.get(ClassName.get(TypeAdapter.class),
-                                                              typeVariableName));
+        String className = FileGenUtils.desanitizeCode(mInfo.getTypeAdapterClassName());
```

Comment:
It seems a little odd that you are calling ```sanitizeCode``` in ```getTypeAdapterClassName``` and then calling ```desanitizeCode``` when using it here.